frontend/public/components/start-guide.tsx

```diff
@@ -10,11 +10,17 @@ import { createProjectMessageStateToProps } from '../reducers/ui';
 import { Disabled, HintBlock, ExternalLink, openshiftHelpBase, LinkifyExternal } from './utils';
 import { connectToFlags } from '../reducers/connectToFlags';
 import { ProjectModel } from '../models';
-import { createProjectModal } from './modals/create-namespace-modal';
+import { createProjectModal } from './modals';
+import { useActivePerspective } from '@console/dynamic-plugin-sdk';
+import { K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
This can be a relative import `../module/k8s/types`

```suggestion
import { K8sResourceKind } from '../module/k8s/types';
```

Please also move the both other `@console/...` imports up the other `import { FLAGS } from '@console/shared/src/constants';` import